app/components/Documentation/components/Pagination/ButtonPaginationDesktop.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { css } from '@emotion/react';
+import ArrowDownIcon from 'icons/ArrowDown';
+import { ButtonPaginationProps } from '../../types';
+
+const ButtonPaginationDesktop: React.FC<ButtonPaginationProps> = ({
+  next,
+  onClickPaginationArticle,
+  getcurrentArrayTitle,
+  NEXT_TYPE_BUTTON,
```

Comment:
not props, use import

Reply:
done: https://github.com/peeranha/peeranha-web/pull/1255/commits/303a47c06288eb7a7876b1135fe75d5894b3359f